test/integration/users.test.js

```diff
@@ -585,6 +575,37 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users/?id", function () {
+    it("Should return given user by id", function (done) {
+      chai
+        .request(app)
+        .get(`/users/?id=${userId}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("User returned successfully!");
```

Comment:
also add the check on phone number and email not being present, as in the older test cases of `users/userId/:id`

Reply:
This is not being added Ujjawal

Reply:
@vikhyat187 updated test for checking that phone and email should not be present and deleted phone and email from user.